### PR TITLE
[DOCU-2849] Update: Introduction section

### PR DESCRIPTION
**Pull request by Guaris**
This PR updates the navigation bar according to the IA rework spreadsheet. 

https://konghq.atlassian.net/browse/DOCU-2849

Depends on this PR to fix the 404 error
https://github.com/kumahq/kuma-website/pull/1257

### CODE REVIEW

File: app/_redirects [hunk 1]
```diff
@@ -36,9 +36,6 @@ https://traffic.kuma.io/exclude-inbound-ports /docs/LATEST_RELEASE/reference/kub
 https://traffic.kuma.io/exclude-outbound-ports /docs/LATEST_RELEASE/reference/kubernetes-annotations/#traffic-kuma-io-exclude-outbound-ports/ 302
 
 
-
-
 # Community section
 
-/contribute/introduction/*  /docs/LATEST_RELEASE/community/contribute-to-kuma/:splat 302
 /reference/license/   /docs/LATEST_RELEASE/community/license/ 302
```
Comment (Guaris) [inline app/_redirects:41]:
```suggestion
/reference/license/   /docs/LATEST_RELEASE/community/license/ 302

```

**Comment by cloudjumpercat** [comment]
@Guaris WRT redirects, see the conversation I had with Fabian in this comment and the comments below it: https://github.com/kumahq/kuma-website/pull/1245#issuecomment-1469730216

TL;DR, I removed all the redirects from my PR that related to URLs that would be changing in 2.2 and moved them to [this JIRA ticket](https://konghq.atlassian.net/browse/DOCU-3048). I'm guessing this will break internal links in docs that are in the `dev` version because they will be pointing to old URLs that have no redirect. So we'll have to conditionally render any of those links?
